travis-meta.sh

```diff
@@ -12,8 +12,13 @@ export PATH=/opt/cabal/head/bin:$PATH
 # Currently doesn't work because Travis uses --depth=50 when cloning.
 #./Cabal/misc/gen-authors.sh > AUTHORS
 
+timed cabal update
+
 # Regenerate files
 timed make gen-extra-source-files
+timed make spdx
```

Comment:
:+1: 

Would be nice to fix `make lexer` as well.

Reply:
I tried, it failed on travis (there were whitespace difference, no idea why)

Reply:
Different version of `alex` or some of its dependencies maybe? Though I have 3.1.7 from @hvr's PPA installed and the diff after `make lexer` is empty.